src/windows-emulator/registry/registry_manager.cpp

```diff
@@ -114,10 +114,8 @@ std::optional<registry_key> registry_manager::get_key(const utils::path_key& key
     return {std::move(reg_key)};
 }
 
-std::optional<registry_value> registry_manager::get_value(const registry_key& key, std::string name)
+std::optional<registry_value> registry_manager::get_value(const registry_key& key, const std::string& name)
```

Comment:
you can use std::string_view here:

```suggestion
std::optional<registry_value> registry_manager::get_value(const registry_key& key, std::string_view name)
```